tahoe_sites/models.py

```diff
@@ -0,0 +1,62 @@
+import uuid
+
+from django.conf import settings
+from django.db import models
+from organizations.models import Organization
+
+
+class UserOrganizationMapping(models.Model):
+    """
+    User membership in an organization.
+
+    Tahoe's fundamental multi-site relationship.
+    """
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    is_active = models.BooleanField(default=True)
+    is_amc_admin = models.BooleanField(default=False)
+
+    class Meta:
+        app_label = 'tahoe_sites'
```

Comment:
Allow using this package before migrating the tables.

```suggestion
        app_label = 'tahoe_sites'
        
        if settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS']:
            # TODO: Remove once we migrate off edx-organizations tables
            managed = False
            db_table = 'organizations_userorganizationmapping'
```